app/pages/verificationwithpassword/verificationwithpassword.js

```diff
@@ -58,6 +59,12 @@ export let VerificationWithPassword = React.createClass({
     this.setState({ loading: false });
   },
 
+  componentDidMount: function() {
+    if (this.props.trackMetric) {
+      this.props.trackMetric('VCA Home Verification - Screen Displayed');
```

Comment:
Every call to `trackMetric` I could find passed in a string. Couldn't find any list of metrics constants.